tensorflow/python/keras/regularizers.py

```diff
@@ -192,6 +194,24 @@ def get_config(self):
     """
     raise NotImplementedError(str(self) + ' does not implement get_config()')
 
+  def _check_penalty_number(self, x):
```

Comment:
I think this doesn't necessary to be an instance method. It can be just a util function.

Reply:
OK! I will change that part.